website/templates/emails/email_contributors_of_public_registrations.txt.mako

```diff
@@ -0,0 +1,12 @@
+Hello ${user.fullname},
+
+Sara and Brian N., please revise below.
```

Comment:
This hotfix removes the option to make a public registrations private. This email informs users with a public registration of the change. 

The longer email details this change as well as the embargoes/retractions change. 

Should the longer email go out now? From a user's perspective, I think 1 email explaining the changes is preferable. We would have to set a date for embargoes/retractions, and @HarryRybacki's PR is still awaiting code review so a firm date may be tricky to pin down. Need weigh in from @sloria @JeffSpies